environment.yml

```diff
@@ -0,0 +1,269 @@
+  - get_terminal_size=1.0.0=h7520d66_0
+  - gettext=0.19.8.1=h15daf44_3
+  - gevent=1.3.0=py36h1de35cc_0
+  - glib=2.56.1=h35bc53a_0
+  - glob2=0.6=py36h94c9186_0
+  - gmp=6.1.2=hb37e062_1
+  - gmpy2=2.0.8=py36hf9c35bd_2
+  - graphite2=1.3.11=h2098e52_2
+  - greenlet=0.4.13=py36h1de35cc_0
+  - h5py=2.7.1=py36ha8ecd60_2
+  - harfbuzz=1.7.6=h171471c_1
+  - hdf5=1.10.2=hfa1e0ec_1
+  - heapdict=1.0.0=py36_2
+  - html5lib=1.0.1=py36h2f9c1c0_0
+  - icu=58.2=h4b95b61_1
+  - idna=2.6=py36h8628d0a_1
+  - imageio=2.3.0=py36_0
+  - imagesize=1.0.0=py36_0
+  - intel-openmp=2018.0.0=8
+  - ipykernel=4.8.2=py36_0
+  - ipython=6.4.0=py36_0
+  - ipython_genutils=0.2.0=py36h241746c_0
+  - ipywidgets=7.2.1=py36_0
+  - isort=4.3.4=py36_0
+  - itsdangerous=0.24=py36h49fbb8d_1
+  - jasper=2.0.14=h636a363_1
+  - jbig=2.1=h4d881f8_0
+  - jdcal=1.4=py36_0
+  - jedi=0.12.0=py36_1
+  - jinja2=2.10=py36hd36f9c5_0
+  - jpeg=9b=he5867d9_2
+  - jsonschema=2.6.0=py36hb385e00_0
+  - jupyter=1.0.0=py36_4
+  - jupyter_client=5.2.3=py36_0
+  - jupyter_console=5.2.0=py36hccf5b1c_1
+  - jupyter_core=4.4.0=py36h79cf704_0
+  - jupyterlab=0.32.1=py36_0
+  - jupyterlab_launcher=0.10.5=py36_0
+  - kiwisolver=1.0.1=py36h792292d_0
+  - lazy-object-proxy=1.3.1=py36h2fbbe47_0
+  - libcurl=7.60.0=hf30b1f0_0
+  - libcxx=4.0.1=h579ed51_0
+  - libcxxabi=4.0.1=hebd6815_0
+  - libedit=3.1.20170329=hb402a30_2
+  - libffi=3.2.1=h475c297_4
+  - libgfortran=3.0.1=h93005f0_2
+  - libiconv=1.15=hdd342a3_7
+  - libopencv=3.4.1=h0f2e407_1
+  - libopus=1.2.1=h169cedb_0
+  - libpng=1.6.34=he12f830_0
+  - libprotobuf=3.5.2=h2cd40f5_0
+  - libsodium=1.0.16=h3efe00b_0
+  - libssh2=1.8.0=h322a93b_4
+  - libtiff=4.0.9=hcb84e12_1
+  - libvpx=1.7.0=h378b8a2_0
+  - libxml2=2.9.8=hab757c2_1
+  - libxslt=1.1.32=hb819dd2_0
+  - llvmlite=0.23.1=py36hc454e04_0
+  - locket=0.2.0=py36hca03003_1
+  - lxml=4.2.1=py36h7166777_0
+  - lzo=2.10=h362108e_2
+  - markupsafe=1.0=py36h3a1e703_1
+  - matplotlib=2.2.2=py36ha7267d0_0
+  - mccabe=0.6.1=py36hdaeb55d_0
+  - mistune=0.8.3=py36h1de35cc_1
+  - mkl=2018.0.2=1
+  - mkl-service=1.1.2=py36h7ea6df4_4
+  - mkl_fft=1.0.1=py36h917ab60_0
+  - mkl_random=1.0.1=py36h78cc56f_0
+  - more-itertools=4.1.0=py36_0
+  - mpc=1.0.3=h7a72875_5
+  - mpfr=3.1.5=h711e7fd_2
+  - mpmath=1.0.0=py36hf1b8295_2
+  - msgpack-python=0.5.6=py36h04f5b5a_0
+  - multipledispatch=0.5.0=py36_0
+  - navigator-updater=0.2.1=py36_0
+  - nbconvert=5.3.1=py36h810822e_0
+  - nbformat=4.4.0=py36h827af21_0
+  - ncurses=6.1=h0a44026_0
+  - networkx=2.1=py36_0
+  - nltk=3.3.0=py36_0
+  - nose=1.3.7=py36h73fae2b_2
+  - notebook=5.5.0=py36_0
+  - numba=0.38.0=py36h1702cab_0
+  - numexpr=2.6.5=py36h057f876_0
+  - numpy=1.14.3=py36h9bb19eb_1
+  - numpy-base=1.14.3=py36h479e554_1
+  - numpydoc=0.8.0=py36_0
+  - odo=0.5.1=py36hc1af34a_0
+  - olefile=0.45.1=py36_0
+  - opencv=3.4.1=py36h6fd60c2_2
+  - openpyxl=2.5.3=py36_0
+  - openssl=1.0.2o=h26aff7b_0
+  - packaging=17.1=py36_0
+  - pandas=0.23.0=py36h1702cab_0
```

Comment:
why do we need pandas?

Reply:
ok we dont need it can you fix it please? thank you.

Reply:
fixed